Funcky.SourceGenerator.Test/Funcky.SourceGenerator.Test.csproj

```diff
@@ -26,7 +26,9 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\Funcky.SourceGenerator\Funcky.SourceGenerator.csproj" />
+        <ProjectReference Include="..\Funcky.SourceGenerator\Funcky.SourceGenerator.csproj">
+          <Aliases>SourceGeneneratorAlias</Aliases>
```

Comment:
I think this might be needed because of the `InternalsVisibleTo` in  Funcky.SourceGenerator.cs, maybe the InternalsVisibleTo can be removed.

Reply:
Hilarious!